File: Programação/Facul/aula pablo/java/portfolio3/veiculos/Veiculo.java
package veiculos;

public class Veiculo {
    private String numeroSerie;
    protected String marca;
    protected String modelo;

    public String getNumeroSerie() {
        return numeroSerie;
    }

    public void setNumeroSerie(String numeroSerie) {
        this.numeroSerie = numeroSerie;
    }

    public String getMarca() {
        return marca;
    }

    public void setMarca(String marca) {
        this.marca = marca;
    }

    public String getModelo() {
        return modelo;
    }

    public void setModelo(String modelo) {
        this.modelo = modelo;
    }

    protected String exibirDetalhes() {
        return String.format("Número de Série: %s%nMarca: %s%nModelo: %s",
                numeroSerie, marca, modelo);
    }
}

File: Programação/Facul/aula pablo/c#/portfolio_3/veiculos/Veiculo.cs
namespace Veiculos;

public class Veiculo
{
    public string NumeroSerie { get; set; } = string.Empty;

    public string Marca { get; set; } = string.Empty;

    public string Modelo { get; set; } = string.Empty;

    protected virtual string ExibirDetalhes()
    {
        return $"Numero de Serie: {NumeroSerie}\nMarca: {Marca}\nModelo: {Modelo}";
    }
}